src/reducers/recoveryMethods.js

```diff
@@ -0,0 +1,20 @@
+import { handleAction } from 'redux-actions'
+import { loadRecoveryMethods } from '../actions/account'
+
+const initialState = {}
+
+const reducer = (state, event) => {
+    const { error, meta: { accountId }, payload, ready } = event
+
+    if (!ready) return state
+
+    if (error) {
+        console.error('error loading recoveryMethods!', payload)
```

Comment:
nit: this is redundant, should be enough to have logging once in the middleware for executing promises

I would just do `if (!ready || error) return state` above (unless there is need to update state based on error).